src/views/virtualmachines/list/components/VirtualMachineRow/VirtualMachineRowLayout.tsx

```diff
@@ -32,6 +36,9 @@ const VirtualMachineRowLayout: React.FC<
 > = ({ activeColumnIDs, obj, rowData: { ips, isSingleNodeCluster, node, vmim } }) => {
   const selected = isVMSelected(obj);
 
+  const vmName = getName(obj);
```

Comment:
what about usememo here? as if ips, node, vmim, isSingle.. getting values it will run